### PR TITLE
monitoring/dashboards: fix query for platform usage

### PR DESCRIPTION
**Pull request by crawford**
The previous query only matches e2e, e2e-aws, and e2e-aws-upgrade. There
are many other tests that target AWS though (e.g. e2e-aws-operator).

### CODE REVIEW

**Comment by crawford** [comment]
The staged results look good. The new graphs show us teetering on the edge of capacity while the old ones show us _well_ below.

File: cluster/ci/monitoring/build_cop.json [hunk 1]
```diff
@@ -567,28 +567,28 @@
          "steppedLine": false,
          "targets": [
             {
-               "expr": "sum(kube_pod_container_status_running{pod=~\"e2e(-aws(-upgrade)?)?\",container=\"teardown\"})",
+               "expr": "sum(kube_pod_container_status_running{pod=~\"(e2e|(.*-aws(-.*)?))\",container=\"teardown\"})",
```
[the remark this answers is not in the document]
Comment (crawford) [reply cluster/ci/monitoring/build_cop.json]: There are 19 jobs that are just called "e2e" and they are all AWS.

[the remark this answers is not in the document]
Comment (smarterclayton) [reply cluster/ci/monitoring/build_cop.json]: Oooo.... ok.  Those are broken, but I'll handle as a follow up.

**Pull request merged**